contracts/cw-proposal-single/src/utils.rs

```diff
@@ -24,3 +27,27 @@ pub fn get_total_power(deps: Deps, dao: Addr, height: Option<u64>) -> StdResult<
         .query_wasm_smart(dao, &voting::Query::TotalPowerAtHeight { height })?;
     Ok(response.power)
 }
+
+/// Validates that the min voting period is less than the max voting
+/// period. Passes arguments through the function.
```

Comment:
I wonder if it would make sense to also have a "maximum minimum voting period" :smile: . Let me explain:

Imagine someone creating a DAO accidentally sets a minimum voting period of years and ends up locking funds in the treasury. Would there be another way to unlock the funds without voting and waiting the minimum period?

I'm far from being an expert, but it is hard to imagine a situation where the minimum voting period must be higher than 1 month (seems to be more than enough time for anyone to exit in case of DAO takeover). So maybe setting `min` to have a maximum value of 1 month would be a good idea to protect against user mistakes.

Reply:
This is great feedback! I tend to think we should leave this unvalidated. IMO limits like these are often best set in the frontend. Ideally, we want the contracts to be as flexible as possible and the frontend to set sane limits. For example, we currently (in the contracts) allow for passing thresholds that are smaller than 50% - normally, nobody should ever set these and in the frontend we make a fuss about it.